ega-data-api-res/src/main/resources/application.yml

```diff
@@ -11,4 +18,31 @@ RES2:
 KEYSERVER:
   listOfServers: ${KEYS_SERVERS:keys}
 
-hystrix.command.default.execution.timeout.enabled: false
\ No newline at end of file
+spring.jpa.properties.hibernate.default_schema: ${DB_SCHEMA}
```

Comment:
I think from line 21 to line 42 is not needed as in res microservice there is no database call involved

Reply:
Done